circle.yml

```diff
@@ -474,7 +474,7 @@ jobs:
             bin/evmone-statetest ~/tests/GeneralStateTests ~/tests/LegacyTests/Constantinople/GeneralStateTests
       - download_execution_tests:
           repo: ipsilon/tests
-          rev: runtime-stackoverflow-fix
+          rev: update-tests
```

Comment:
Because of the risk of rebase or branch deletion, I try to use tags, e.g. `eof-opcodes-20230619`. But it's up to you in the end.

Reply:
Made a tag